<!-- -*- xml -*- -->

<launch>

<param name="use_sim_time" value="true" />

<node pkg="rosbag" type="play" name="player"  args="--clock /home/vignesh/thorvald_move.bag"/>

<node pkg="thorvald_2d_nav" type="poly_nav_RANSAC" name="poly_nav_RANSAC" output="screen"/>

<node pkg="thorvald_2d_nav" type="poly_nav_line_SLAM" name="poly_nav_line_SLAM" output="screen"/>

<!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find /home/vignesh/catkin_ws/src/thorvald_2d_nav/record_scan_config.rviz"/-->

</launch>
